### DISCUSSION TITLE
Problem using cx-freeze on Linux (Mint 20.3)

### DISCUSSION THREAD

**Discussion by aiotech-pub** [marcelotduarte/cx_Freeze#1579] [Q&A]
I am porting a win32 Python application to Linux and I want to build its executable using cx-freeze. The win32 application works and I am able to build its executable using cx-freeze. In Linux, instead, cx-freeze ends with a strange error:

running build
running build_exe
error: [Errno 2] No such file or directory: '/home/massimo/.local/share/virtualenvs/JanasMagicKey-vnaevMls/lib/python3.8/site-packages/wheel-0.37.1.dist-info/__pycache__

I checked the win32 project and I see there is no pycache folder in its equivalent position so I really do not understand the error itself and what I have to do to solve the problem.
I am working using PyDev and pipenv.
I attached my setup file for Linux (named setup_cx.txt 
[setup_cx.txt](https://github.com/marcelotduarte/cx_Freeze/files/9282386/setup_cx.txt)
instead of setup_cx.py)

**Comment by marcelotduarte** [comment]
Try to use excludes=["wheel"] option in your buildOptions.

**Reply by aiotech-pub** [reply]
no effect, same error as before. I attached the updated setup_c
[setup_cx.txt](https://github.com/marcelotduarte/cx_Freeze/files/9282621/setup_cx.txt)
x

**Comment by marcelotduarte** [comment]
What is the cx_Freeze version?

Try using the option in: https://github.com/marcelotduarte/cx_Freeze#installation

**Reply by aiotech-pub** [reply]
It was cx_Freeze 6.6 I just updated it to 6.11.1 and seems solves the problem we are discussing here. 
I just built the program, it runs but there is an error and a comment saying that Tcl was not properly installed. Take care that the program runs if i launch it inside Eclipse (I am using Eclipse + PyDev on Linux Mint 20.3) so I suppose the problem is related to how I built the Linux esecutable.

**Reply by marcelotduarte** [reply]
if you use tkinter, with 6.10 it should work when tkinter is installed in the machine.
With 6.12 (pre-release) it should work too...

**Reply by marcelotduarte** [reply]
Can you test with the development version?

To install the latest development build:

`pip install --pre --extra-index-url https://marcelotduarte.github.io/packages/ cx_Freeze`

**Reply by marcelotduarte** [reply]
[cx_Freeze 6.12.0](https://github.com/marcelotduarte/cx_Freeze/releases/tag/6.12.0) has just been released.
